gizmos/search.py

```diff
@@ -68,13 +68,13 @@ def get_search_results(
     - property
     - order"""
     names = defaultdict(dict)
-    if text:
+    if search_text:
         # Get labels
         query = sql_text(
             """SELECT DISTINCT subject, value FROM statements
-            WHERE predicate = :label AND value LIKE :text"""
+            WHERE predicate = :label AND lower(value) LIKE :text"""
```

Comment:
If the `:text` includes uppercase characters, this will never match?

Reply:
The text is lowered when the query is executed: https://github.com/ontodev/gizmos/pull/94/files#diff-71015c9edb7bf1c92a0876a0329d46c23b8321b5782c7c9dccfd236c806a9820R77

Reply:
It can be if you want me to update it. It gets bound on L77.

We have a test for search: https://github.com/ontodev/gizmos/blob/master/tests/test_search.py
I'll add some mixed case stuff into there.

Reply:
Now I see. Not need to rename. Yes, please add some case-insensitive tests.